packages/babel-plugin-transform-classes/test/fixtures/assumption-superIsCallableConstructor/invalid-extend-builtins-Iterator/exec.js

```diff
@@ -0,0 +1,3 @@
+class RangeIterator extends Iterator {}
+
+expect(() => new RangeIterator).toThrow("Constructor Iterator requires 'new'");
```

Comment:
I can execute it in chrome.
![image](https://github.com/user-attachments/assets/b5f292f1-1e89-4d4f-91e7-f1d0068b4dd5)

Reply:
Oops I didn't notice the assumption, sorry!